demos/internal-developers/graph-resource-usage.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>A possible extension is to add reddit to our datasets code, but not provide automatic download?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
That won't directly help here, because this needs to be manually reading the files, but it would help for potential future demos that might use it.